### ISSUE TITLE
AssertionError: duplicated lora name: lora_unet_down_blocks_0_attentions_0_transformer_blocks_0_attn1_to_q

### ISSUE THREAD

**Issue by wylqq312715289** [bmaltais/kohya_ss#462]
AssertionError: duplicated lora name: lora_unet_down_blocks_0_attentions_0_transformer_blocks_0_attn1_to_q

```
accelerate launch --num_cpu_threads_per_process=2 "train_network.py" \
  --pretrained_model_name_or_path="runwayml/stable-diffusion-v1-5" \
  --train_data_dir="/ssd3/exec/aabbcc/Lora/kohya_ss/train_datasets/images" \
  --resolution=512,512 \
  --output_dir="/ssd3/exec/aabbcc/Lora/kohya_ss/outputs" \
  --logging_dir="/ssd3/exec/aabbcc/Lora/kohya_ss/outputs/trainlog" \
  --network_alpha="128" \
  --save_model_as=safetensors \
  --network_module=networks.lora \
  --text_encoder_lr=5e-5 \
  --unet_lr=0.0001 \
  --network_dim=128 \
  --output_name="guofeng-dilay" \
  --lr_scheduler_num_cycles="1" \
  --learning_rate="0.0001" \
  --lr_scheduler="constant" \
  --train_batch_size="2" \
  --max_train_steps="2200" \
  --save_every_n_epochs="1" \
  --mixed_precision="fp16" \
  --save_precision="fp16" \
  --seed="1234" \
  --caption_extension=".txt" \
  --cache_latents \
  --optimizer_type="AdamW8bit" \
  --max_data_loader_n_workers="1" \
  --clip_skip=2 \
  --bucket_reso_steps=64 \
  --bucket_no_upscale
```

**Comment by bmaltais** [comment]
Was it working in a previous release? Can you paste the traceback for the issue?

**Comment by lawfordp2017** [comment]
I got this error message as well and it seems to be getting confused with:
  UNET_TARGET_REPLACE_MODULE = ["Transformer2DModel", "Attention"]
as Attention actually exists inside of Transformer2DModel.

I added explicit memos in the create_modules routine:
```

    def create_modules(prefix, root_module: torch.nn.Module, target_replace_modules) -> List[LoRAModule]:
      loras = []

      parent_memo = set()
      child_memo = set()
      for name, module in root_module.named_modules(memo=parent_memo):
        if module in child_memo:
          continue
        if module.__class__.__name__ in target_replace_modules:
          # TODO get block index here
          for child_name, child_module in module.named_modules(memo=child_memo):

```

**Comment by liushanyuan18** [comment]
This issue is related to the diffuser version, which can be resolved using the official 0.14.0 version

**Comment by 303251669** [comment]
> This issue is related to the diffuser version, which can be resolved using the official 0.14.0 version

You are right,think you!
